### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by abd-ar**
- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

### CODE REVIEW

**Comment by DNGSlug** [comment]
hi there

i do apologise i as i am absolutely new on that platform and i am doing 90% things by guess after reading

so i am sorry for that and please keep guiding me

Many Regards
Naveed

**Comment by DNGSlug** [comment]
it's an honest mistake
and i do offer my unconditional apology for this

many thanks for your guidance

**Comment by oneemo95** [comment]
[Uploading Octocats.zip…]()

**Review by kkathawut — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
 ---
 layout: index
-lang: en
```
Comment (MaryJaneLove4) [inline index.html:3]: Angličtina

**Review by MaryJaneLove4 — APPROVED** [review]
Anglický jazyk

**Review by UttaM1990 — APPROVED** [review]

**Review by oneemo95 — APPROVED** [review]

**Review by UttaM1990 — APPROVED** [review]

**Review by MaryJaneLove4 — APPROVED** [review]
👍

**Review by MaryJaneLove4 — APPROVED** [review]

**Pull request closed without merging**